pipeline/attestation/selftest/run-selftest.sh

```
@@ -0,0 +1,83 @@
+[[ $? -ne 0 ]] && ok "T4 attestation 篡改 → 红（digest 不符+验签必坏双锚）" || bad "T4 attestation 篡改漏检"
+
+# T5 错公钥（第二把钥匙验签）
+openssl genrsa -out "$TMP/sk2.pem" 2048 2>/dev/null
+openssl rsa -in "$TMP/sk2.pem" -pubout -out "$TMP/pub2.pem" 2>/dev/null
+run_verify --bundle "$TMP/bundle" --artifact "$TMP/artifact.tar.gz" --pubkey "$TMP/pub2.pem" >/dev/null 2>&1
+[[ $? -ne 0 ]] && ok "T5 错公钥 → 红（RS256 验签拒）" || bad "T5 错公钥漏检"
+
+# T6 缺件（bundle 目录空）→ exit 2 infra；--expect-commit 不符 → 红
+mkdir -p "$TMP/bundle-empty"
+run_verify --bundle "$TMP/bundle-empty" --artifact "$TMP/artifact.tar.gz" --pubkey "$TMP/pub.pem" >/dev/null 2>&1
+[[ $? -eq 2 ]] && ok "T6 缺 bundle → exit 2（fail-closed infra）" || bad "T6 缺件退出码错（$?）"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)